### ISSUE TITLE
[Bug]: Why Nova Video Player needs to record audio (needs microphone permission) ? 

### ISSUE THREAD

**Issue by tp01-gogogo** [nova-video-player/aos-AVP#902]
### Problem description

Why Nova Video Player needs to record audio? And if I don't allow, an error will occur.

### Steps to reproduce the issue

Storage permission has been given, microphone permission has not.
![Screenshot_20230615-102410_Permission controller](https://github.com/nova-video-player/aos-AVP/assets/111100869/e7077f1c-2fe4-4485-9d31-1f25409ee570)
![Screenshot_20230615-102635_Nova Video Player](https://github.com/nova-video-player/aos-AVP/assets/111100869/749b13e1-6d02-474c-9d5c-1efee9cb5f57)
![Screenshot_20230615-103231_Permission controller](https://github.com/nova-video-player/aos-AVP/assets/111100869/41604dfa-a4b3-4b34-af7c-d33cbe3bf582)

### Your phone/tablet/androidTV model

LineageOS 20 on mobile phone

### Operating system version

Android 13

**Comment by mschumacher69** [comment]
So you can search by voice I believe.

**Comment by courville** [comment]
Indeed, search by voice is the motivation though we should be able not to agree (to be implemented). And voice search is not working for all yet it seems.
